### ISSUE TITLE
Flows should be posted as files

### ISSUE THREAD

**Issue by johnwalicki** [ibm-early-programs/watson-speech-sockets#7]
Instead of embedding the Node-RED flows inside the README.md, it would be better to post them to GitHub as files.

**Comment by chughts** [comment]
You mean like the documentation - https://github.com/watson-developer-cloud/node-red-labs/tree/master/starter-kits/dictaphone
https://github.com/watson-developer-cloud/node-red-labs/tree/master/starter-kits/babelfish
and
https://github.com/watson-developer-cloud/node-red-labs/tree/master/starter-kits/text_to_speech_audio

**Comment by johnwalicki** [comment]
Yes - If you add links to the documentation into the README.md on this page, that would be helpful. There's another ticket about documentation. This ticket requests Remove / Replace the embedded flows in the README with real files.

FWIW - All of the Node-RED flows, documentation and screenshots would benefit from removal of Function nodes that can be replaced by Change nodes.
